wetmap/src/components/newModals/setting/index.tsx

```diff
@@ -1,79 +1,47 @@
 
-import SettingsView from './view';
-
+import { signOut } from '../../../supabaseCalls/authenticateSupabaseCalls';
 import { SessionContext } from '../../contexts/sessionContext';
-import { useContext, useState, useEffect } from 'react';
+import { UserProfileContext } from '../../contexts/userProfileContext';
+import SettingsView from './view';
 import React from 'react';
+import { useContext } from 'react';
+import PartnerAccountRequestModal from '../../modals/partnerAccountRequestModal';
+import { ModalContext } from '../../reusables/modal/context';
 
 
 export default function Settings(props) {
   const { activeSession, setActiveSession } = useContext(SessionContext);
```

Comment:
we only need the setter in this case so you can remove activeSession to resolve the error